app/modules/annotations/parameters.py

```diff
@@ -18,8 +21,18 @@ class CreateAnnotationParameters(Parameters, schemas.BaseAnnotationSchema):
         required=False,
     )
 
-    class Meta(schemas.DetailedAnnotationSchema.Meta):
-        pass
+    class Meta(schemas.BaseAnnotationSchema.Meta):
+        fields = schemas.BaseAnnotationSchema.Meta.fields + (
+            Annotation.ia_class.key,
+            Annotation.bounds.key,
+        )
+
+    @validates_schema
+    def validate_bounds(self, data):
```

Comment:
Is this some magic foo that automatically gets called. I couldn't see where it was called from.

Reply:
i believe so, yes.  but i was just following users/parameters.py tbh.  looks to be some magic via marshmallow library.